docs/book/src/reference/jobs.md

```diff
@@ -76,7 +76,9 @@ Prow Periodics:
   * GINKGO_FOCUS: `[K8s-Upgrade]`
 * [periodic-cluster-api-e2e-workload-upgrade-1-25-1-26-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `stable-1.26`
```

Comment:
Please drop "periodic-cluster-api-e2e-workload-upgrade-1-21-1-22-main" above

(also the link below)

Reply:
Thanks, updated